src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -204,15 +204,15 @@ public void updateFilteredIncomeList(Predicate<Entry> predicate) {
             throw new AssertionError("This method should not be called.");
         }
 
-        @Override
-        public void updateExpensePieChart() {
-            throw new AssertionError("This method should not be called.");
-        }
-
-        @Override
-        public void updateIncomePieChart() {
-            throw new AssertionError("This method should not be called.");
-        }
+        // @Override
```

Comment:
This needs to wait for `ViewCommand` to work in another PR combined with https://github.com/AY2223S1-CS2103T-W17-2/tp/issues/52